taskcluster/scripts/slack_notifier.py

```diff
@@ -0,0 +1,135 @@
+import json
```

Comment:
Same as above. As well since this lives in `taskcluster/scripts` I would add how this file can be used if people would like to start using it.